pkg/virt-config/feature-gates.go

```diff
@@ -44,6 +44,7 @@ const (
 	DownwardMetricsFeatureGate = "DownwardMetrics"
 	NonRoot                    = "NonRootExperimental"
 	ClusterProfiler            = "ClusterProfiler"
+	LaunchSecurityGate         = "LaunchSecurity"
```

Comment:
Can we call this `LaunchSecuritySEV`? I don't want to spend too much time on this, but maybe `WorkloadEncryptionSEV`?

Reply:
Initially I called it `LaunchSecurity` just because it maps to the libvirt naming in domxml. But yeah, I can rename it. `WorkloadEncryptionSEV` seems also good for me.

Reply:
Awesome :+1: